app/models/broadcast.rb

```diff
@@ -10,6 +10,8 @@ class Broadcast < ApplicationRecord
   validates :banner_style, inclusion: { in: VALID_BANNER_STYLES }, allow_blank: true
   validate  :single_active_announcement_broadcast
 
+  before_save :update_last_active_at, if: :will_save_change_to_active?
```

Comment:
Doesn't look like `will_save_change_to_active` is being used at all here

Reply:
Huh, it looks like you're right! I just removed the method and it didn't result in any errors - strange. 🤔  I have updated the model to not include this dead code!